plugins/pencilblue/controllers/actions/salesforce/register.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const request = require('request-promise');
+
+module.exports = function RegisterSFControllerModule(pb) {
+    /**
+     * Registers a user via Salesforce
+     * @class RegisterSFControllerModule
+     * @constructor
+     */
+
+    const SalesforceStrategyService = require('../../../services/salesforce/salesforce_strategy_service')(pb);
+    class RegisterSFController extends pb.BaseController {
+        render(cb) {
+            this.sanitizeObject(this.body);
+            this.salesforceRegister(cb);
+        }
+
+        async salesforceRegister(cb) {
+            const salesforceStrategyService = new SalesforceStrategyService();
+            let response = await salesforceStrategyService.register(this.req);
+
+            if (response && response.enableCustomRegister && response.url) {
```

Comment:
Minor: 
I don't think we need dual settings here. 
One is enough, response.url

Reply:
Yes, I can see your point. But this is part of the acceptance criteria provided by the card. So I'd like to hear what @bradleylowenstein has to say about it

Reply:
Daniel originally had 2 settings so that you can turn this on and off and still keep the value if the setting is off in case it ever needed to be turned on again.  I would prefer to have both settings to be consistent with the login options.